google/cloud/aiplatform/utils/__init__.py

```diff
@@ -591,3 +594,120 @@ def _timestamped_copy_to_gcs(
 
     gcs_path = "".join(["gs://", "/".join([blob.bucket.name, blob.name])])
     return gcs_path
+
+
+def upload_to_gcs(
```

Comment:
I've moved the new functions to `utils/gcs_utils.py` module. We can move other functions to that module in another refactoring PR so that we can keep the PR scope focused.